Orange/data/io.py

```diff
@@ -915,12 +921,13 @@ def read(self):
         raise ValueError('Cannot parse dataset {}: {}'.format(self.filename, error)) from error
 
     @classmethod
-    def write_file(cls, filename, data):
+    def write_file(cls, filename, data, with_annotations=True):
         with cls.open(filename, mode='wt', newline='', encoding='utf-8') as file:
             writer = csv.writer(file, delimiter=cls.DELIMITERS[0])
-            cls.write_headers(writer.writerow, data)
+            cls.write_headers(writer.writerow, data, with_annotations)
             cls.write_data(writer.writerow, data)
-        cls.write_table_metadata(filename, data)
+        if with_annotations:
```

Comment:
I would remove this `if` - saving the additional metadata file is not related to type annotations for variables.